### PR TITLE
Maintenance PR

### PR DESCRIPTION
**Pull request by roverflow**
##### SUMMARY

Uses [https://github.com/ansible/ansible-content-actions](https://github.com/ansible/ansible-content-actions)

##### ISSUE TYPE

- CI Pull Request

##### ADDITIONAL INFORMATION

```paste below
ack -> check_label
push -> draft_release ( manual only )
```

### CODE REVIEW

**Review by Ruchip16 — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+paramiko
```
Comment (KB-perByte) [inline requirements.txt]: why?

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -15,32 +15,35 @@ on:  # yamllint disable-line rule:truthy
...
         git+https://github.com/ansible-collections/ansible.netcommon.git
   all_green:
-    if: ${{ always() && (github.event_name != 'schedule') }}
+    if: ${{ always() }}
     needs:
       - changelog
       - sanity
       - unit-galaxy
-      - unit-source
+      - ansible-lint
```
Comment (KB-perByte) [inline .github/workflows/tests.yml:38]: needs unit source too!

File: .github/workflows/draft_release.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+name: "Draft release"
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+on:  # yamllint disable-line rule:truthy
+  workflow_dispatch:
+env:
+  NAMESPACE: ${{ github.repository_owner }}
+  COLLECTION_NAME: ios
```
Comment (KB-perByte) [inline .github/workflows/draft_release.yml]: this to be updated

File: .ansible-lint [hunk 4]
```diff
@@ -1,5 +1,7 @@
 ---
+
 profile: production
 
-exclude_paths:
-  - changelogs/changelog.yaml
+# exclude_paths:
```
Comment (KB-perByte) [inline .ansible-lint]: can remove! no need to comment it out!

**Review by KB-perByte — CHANGES_REQUESTED** [review]

File: test-requirements.txt [hunk 5]
```diff
@@ -1,9 +1,9 @@
 # For ansible-tox-linters
-black==23.3.0 ; python_version >= '3.7'
+black==23.3.0
 flake8
```
Comment (Ruchip16) [inline test-requirements.txt]: i think flake8 should be removed from here as well

File: mypy.ini [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+show_error_codes = true
+show_error_context = true
+strict = true
+strict_optional = true
+
+[mypy-ansible.*]
+# No type hints as of version 2.12
+ignore_missing_imports = true
+
+[mypy-ansible_collections.cisco.ios.*]
```
Comment (Ruchip16) [inline mypy.ini]: instead of `cisco.ios` it should be` [mypy-ansible_collections.splunk.es.*]`

**Review by NilashishC — APPROVED** [review]

**Pull request merged**